index.js

```diff
@@ -58,6 +58,10 @@ function prettyFactory (options) {
   const singleLine = opts.singleLine
 
   const colorizer = colors(opts.colorize)
+  if (opts.colorize === true) {
+    coloretteOptions.enabled = true
```

Comment:
@jorgebucaran Are you open to a PR for a new semver major which would introduce stateless colourer instead of singleton, or that was a very deliberate design choice? Alternatively I can add this as an alternative opt-in API.

Reply:
Why are we using `.enabled = true` in the first place? 

This exists mostly as an escape hatch to automatic color detection. You shouldn't need it, really.

Reply:
@jorgebucaran How else can you propagate explicit forcing of colour from pino-pretty to colorette?

Reply:
> How else can you propagate explicit forcing of colour from pino-pretty to colorette?

Oh, yes, you'd use the escape hatch for that (programmatically force the color on or off).

Just curious, how were you doing this with chalk?

Reply:
@jorgebucaran I might be wrong in understanding how chalk works, but the way I read original code, it was the other way around, we were reading chalk config and disabling it internally if it said that colourizing is not supported. Does chalk shut off colourizing internally as well if it thinks that it is not supported? If yes, we can pretty much replicate same behaviour without additional configuration.

Reply:
You could do the same via colorette's `options.enabled` (it tells you if your terminal supports color or not). 

That's what [chalk.supportsColor](https://github.com/chalk/chalk#supportscolor) essentially does.